indico/modules/events/layout/models/menu.py

```diff
@@ -27,6 +28,30 @@ def _get_next_position(context):
     return (res[0] or 0) + 1
 
 
+def _is_speaker(event, user):
```

Comment:
I just saw that we have `is_user_registered` as a method in the `Event` model... so maybe moving this there as a `is_user_speaker` method isn't a bad idea :)